### ISSUE TITLE
DayLayout.getDayView returning wrong DayView when first time invoked

### ISSUE THREAD

**Issue by TKolbusz** [recruit-mp/LightCalendarView#14]
As title says, 
Logcat 
// first invocation
D/DayLayout: setSelectedDay Mon Jan 02 12:55:21 GMT 2017
D/DayLayout: setSelectedDay2 DayView(Sun Jan 01 12:55:21 GMT 2017)
//2nd invocation
D/DayLayout: setSelectedDay Mon Jan 02 12:55:21 GMT 2017
D/DayLayout: setSelectedDay2 DayView(Mon Jan 02 12:55:21 GMT 2017)

DayLayout.kt

```
    fun setSelectedDay(date: Date) {
        Log.d("DayLayout", "setSelectedDay " + date.toString())
        setSelectedDay(getDayView(date))
    }

    fun getDayView(date: Date): DayView? = childList.getOrNull(date.daysAfter(firstDate.time).toInt()) as? DayView

    private fun setSelectedDay(view: DayView?) {
        Log.d("DayLayout","setSelectedDay2 " + view.toString())
        selectedDayView?.apply {
            isSelected = false
            updateState()
        }
        selectedDayView = view?.apply {
            isSelected = true
            updateState()
            callback?.onDateSelected(date)
        }
    }
```

**Comment by recruit-mahayash** [comment]
Hi @TKolbusz,

Thanks a lot for the issue and further investigations! We are now working on enhancements and bug-fixes including this in our private repository. The next version will be soon released.

Thank you.

**Comment by recruit-mahayash** [comment]
@TKolbusz,

While developing [the release version 1.0.1](https://github.com/recruit-mp/LightCalendarView/releases/tag/1.0.1), we have tested calling the same method in the same situation, but the problem could not be reproduced. This issue may be somehow solved in [Release 1.0.1](https://github.com/recruit-mp/LightCalendarView/releases/tag/1.0.1). Could you confirm if the same problem occurs with the latest library version?

Thanks!

**Comment by TKolbusz** [comment]
@recruit-mahayash Yep it works well in new release, the issue that I explained was really weird, it actually fixed itself by mistake, it was probably unreproducable because it was caused by my other changes in code.
